src/components/FeaturePanel/Properties/IdSchemaFields.tsx

```diff
@@ -30,11 +30,6 @@ const render = (uiField: UiField, feature: Feature): string | ReactNode => {
     return renderValue('wikidata', feature.tags.wikidata);
   }
 
-  // combo with options
-  if (fieldTranslation?.options?.[v]) {
-    return renderValue(k, fieldTranslation.options[v]?.title);
-  }
```

Comment:
I think this case can also occur sometimes. Can you please check it? Or say so, and i will try to find it. 

This code is quite smelly 😅

Reply:
👌